conda/gateways/connection/session.py

```diff
@@ -156,6 +157,26 @@ def __init__(self, auth: AuthBase | tuple[str, str] | None = None):
 
         self.proxies.update(context.proxy_servers)
 
+        ssl_context = None
+        if context.ssl_verify == "truststore":
+            try:
+                import ssl
+
+                import truststore
+
+                ssl_context = truststore.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+            except ImportError:
+                warnings.warn(
```

Comment:
See below for my remark about the need to bubble up the error here, if users specifically configure to use `truststore`, we need to raise a `CondaError` or something instead of assuming ssl_verify=True (e.g. they may prefer to configure a custom cert bundle instead).

Reply:
I raise a `CondaError` now